services/api/src/routes/auth.js

```diff
@@ -135,28 +135,25 @@ router
       try {
         await verifyLoginAttempts(authUser);
       } catch (error) {
-        await AuditEntry.append('Reached max authentication attempts', ctx, {
+        await AuditEntry.append('Reached max authentication attempts', {
+          ctx,
           category: 'security',
-          object: authUser,
-          user: authUser.id,
```

Comment:
we only care about the user id ... not sure how i feel like sending down the full object ...

Reply:
the id will get pulled out so at the end it will only be the id